File: app/src/main/java/de/markusfisch/android/shadereditor/activity/PreferencesActivity.java
package de.markusfisch.android.shadereditor.activity;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import de.markusfisch.android.shadereditor.fragment.PreferencesFragment;

public class PreferencesActivity extends AbstractContentActivity {
	@NonNull
	@Override
	protected Fragment defaultFragment() {
		return new PreferencesFragment();
	}
}

File: app/src/main/java/de/markusfisch/android/shadereditor/activity/PreferencesActivity.kt
package de.markusfisch.android.shadereditor.activity

import androidx.fragment.app.Fragment
import de.markusfisch.android.shadereditor.fragment.PreferencesFragment

class PreferencesActivity : AbstractContentActivity() {
    override fun defaultFragment(): Fragment {
        return PreferencesFragment()
    }
}
